api/src/main/java/jakarta/annotation/ManagedBean.java

```diff
@@ -29,6 +29,7 @@
  */
 @Target(TYPE)
 @Retention(RUNTIME)
+@Deprecated(/*since = "10.0", forRemoval = true*/)
```

Comment:
don't we want to have `since="10",  forRemoval="true"` there? Is there a need to keep compatibility with SE 8 in this project? (I do not think so)

Reply:
I think the version should be the version for common annotations not profile release version. It needs to be 
```suggestion
@Deprecated(/*since = "2.1.1", forRemoval = true*/)
```

Reply:
Why, what is the correct version? 10.0 would be the version it is deprecated in for future removal. That is how the SecurityManager depreciation was done for example:
```java
@Deprecated(since="17", forRemoval=true)
public class SecurityManager {
```

Reply:
Ok, I see, you are talking about the common annotations version. Fixed.